client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -51,11 +57,16 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
     sortFieldOptions: Array<{label: string, onSelect(): void}>
     container?: HTMLDivElement;
     resizeObserver: ResizeObserver;
+    userList: Array<{label: string, onSelect(): void}>
 
     constructor(props) {
         super(props);
+        this.state = {
+            name: ''
+        };
         const {gettext} = superdeskApi.localization;
 
+        this.props.users.unshift({_id: null, display_name: 'All'});
```

Comment:
Props should **never** be modified, no matter what the circumstances.
Instead you create a new array in the render method, combining this `All` choice and the list of users

Reply:
my bad I forgot to remove this.